tests/queries/0_stateless/01651_map_functions.sql

```diff
@@ -27,3 +29,7 @@ select map( 'aa', 4, 'bb' , 5) as m, mapKeys(m), mapValues(m);
 select map( 'aa', 4, 'bb' , 5) as m, mapContains(m, 'aa'), mapContains(m, 'k');
 
 select map(0, 0) as m, mapContains(m, number % 2) from numbers(2);
+
+select mapFromArrays(['aa', 'bb'], [4, 5]);
```

Comment:
Function doesn't work with full columns, `select mapFromArrays(['aa', 'bb'], materialize([4, 5]));`  or `select mapFromArrays(['aa', 'bb'], [4, number]) from numbers(10);` throws fatal. So, it's good to add cases with full columns .

Reply:
Fixed.